e2e-tests/pages/assignedSitesPage.page.js

```diff
@@ -3,13 +3,12 @@
 class assignedSitesPage {
 
     get assignedSites() { return $('//a[@href="/assigned-sites"]') }
-    get pagehHeader() { return $('//app-assigned-sites//div//div//h2') }
-    get region() { return $('//div//table//tr//td[text()="CCO"]') }
+    get title() { return $('//h2[@id="assignedSitesTitle"]') }
```

Comment:
The getPageHeader function has been renamed getTitle and changed to look for the ID tag in an H2 rather than just any H2 tag.   I can change the function name back to pageHeader if you want?    But this is a different check to the main scenario which is defineStep('the user should be able to view all the {string}', function (assignedSites)  and looks for CCO.

Reply:
I have changed the function name back to pagehHeader, to reduce the number of changes.